examples/anvil/Cargo.toml

```diff
@@ -1,10 +1,14 @@
 [package]
```

Comment:
Can we make `cargo release` not go over the examples crates if possible? Right now it thinks all the examples should be published

Reply:
doesn't `publish = false` do the trick? What's the command you run to release